client/src/components/NationwideMap.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div id='map-container'></div>
+  <div id="map-container"></div>
```

Comment:
My prettier has been making all strings single quote - what config are you using to get double quotes in html elements?

Reply:
Can we keep single?

Reply:
Done with a hacky fix.